### ISSUE TITLE
Pre-commit finding some issue

### ISSUE THREAD

**Issue by enadeau** [stravalib/stravalib#428]
Some flake8 error made their way into the code. I realized that when trying to configure ruff.

```
$ pre-commit run --all
fix end of files.........................................................Passed
check for case conflicts.................................................Passed
trim trailing whitespace.................................................Passed
flake8...................................................................Failed
- hook id: flake8
- exit code: 1

examples/strava-oauth/server.py:2:1: F401 'logging' imported but unused
examples/strava-oauth/server.py:4:1: F401 'flask.jsonify' imported but unused
examples/strava-oauth/server.py:4:1: F401 'flask.redirect' imported but unused
stravalib/__init__.py:1:1: F401 'stravalib.client.Client' imported but unused
stravalib/tests/integration/strava_api_stub.py:132:25: F541 f-string is missing placeholders

black....................................................................Passed
isort....................................................................Passed
```

I don't quite know how this is possible, but I'd assume that it's from before it was well integrated in CI. I don't think we should worry too much about it. 

Incoming PR to fix the errors.

**Comment by enadeau** [comment]
I was not up to date with main after pulling only one error was left to fix in the PR

**Comment by enadeau** [comment]
Was actually fixed in #425
